yarn-error.log

```diff
@@ -0,0 +1,131 @@
+Arguments: 
```

Comment:
It looks like an error log snuck in. Mind removing this from your PR?

Reply:
Woops, done. Added some stuff to `.gitignore` while I was at it